RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -1,10 +1,96 @@
 //
 //  RockPaperScissors - main.swift
-//  Created by yagom. 
+//  Created by Lingo, 우롱차.
 //  Copyright © yagom academy. All rights reserved.
 // 
 
 import Foundation
 
-print("Hello, World!")
+enum Settings {
+    static let exitCode = 0
+}
 
+enum GameDisplayMessage {
+    static let menu = "가위(1), 바위(2), 보(3)! <종료 : 0> : "
+    static let gameDidEnd = "게임 종료"
+    static let playerDidWin = "이겼습니다!"
+    static let playerDidLose = "졌습니다!"
+    static let playerDidDraw = "비겼습니다!"
+    static let invalidPlayerInput = "잘못된 입력입니다. 다시 시도해주세요."
+    static let error = "에러가 발생했습니다. 게임을 다시 실행시켜 주세요."
+}
+
+enum HandType: Int, CaseIterable {
+    case scissor = 1
+    case rock = 2
+    case paper = 3
+    
+    static var randomHandType: Int? {
+        return Self.allCases.randomElement()?.rawValue
```

Comment:
열거형 타입 이름에 이미 `HandType` 단어가 들어가있습니다.
그래서 연산 프로퍼티 네이밍에도 들어가면 중복입니다! 😄 

간단하게 `random` 으로 바꿔보거나, `randomPick` 같은 건 어떨까요~?

```suggestion
    static var random: Int? {
        return Self.allCases.randomElement()?.rawValue
```